src/foam/nanos/auth/Address.js

```diff
@@ -27,31 +27,34 @@ foam.CLASS({
     {
       class: 'String',
       name: 'type',
-      documentation: 'Address type.'
+      documentation: 'Address type. COMMENT: What is this?'
     },
     {
       class: 'Boolean',
       name: 'verified',
-      documentation: 'Identifies if address has been verified.'
+      documentation: 'Verifies that the address exists.'
     },
     {
       class: 'Boolean',
       name: 'deleted',
-      documentation: 'Marks address as deleted.'
+      documentation: 'Indicates the address as deleted.'
```

Comment:
Perhaps: When 'true', Indicates that this address record has been marked as deleted.  To normal privileged users this address record does not exist, but it still exists in the system as historical data may refer to it.  Only an administrator can see deleted records.